q2_feature_classifier/tests/test_cutter.py

```diff
@@ -23,16 +23,18 @@ def setUp(self):
 
     def test_extract_reads(self):
         # extract_reads should generate a FeatureData(Sequence) full of reads
-        length = 75
+        trunc_len = 75
```

Comment:
Can you add some additional tests to cover the [branches introduced](https://github.com/qiime2/q2-feature-classifier/pull/69/files#diff-543f7c49a71dc85a194d9f5fa66946c8R115) in the PR? Basically checking output when `trunc_len` is negative, `trim_left` is negative, and both params are negative (ie not supplied). Those could be new tests, or you could do some kind of loop in this test, I don't have a preference.